common/lib/xmodule/xmodule/course_module.py

```diff
@@ -662,7 +662,14 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
-
+    issue_badges = Boolean(
+        display_name=_("Issue OpenBadges"),
+        help=_(
+            "Issue OpenBadges badges for this course. Badges are generated when certificates are created."
+        ),
+        scope=Scope.settings,
+        default=False
```

Comment:
I could turn it on by default, but that will mean some courses will have old versions of badges floating around that may not fit with their aesthetics. Some may want to wait until customization is available.